docs/src/kernel/linux-compatibility/limitations-on-system-calls/file-systems-and-mount-control.md

```diff
@@ -7,3 +7,73 @@ fdatasync, sync, syncfs, sync_file_range, open_tree, move_mount, fsopen,
 fsconfig, fsmount, and fspick
 under this category.
 -->
+
+## Mount and Unmount File Systems
+
+### `mount`
+
+Supported functionality in SCML:
+
+```c
+// Move the existing mount point atomically
+mount(
+    source, target, filesystemtype,
+    mountflags = MS_MOVE,
+    data
+);
+
+// Create a bind mount
+mount(
+    source, target, filesystemtype,
+    mountflags = MS_BIND | MS_REC | MS_MOVE,
+    data
+);
+```
+
+Silently-ignored mount flags:
+* `MS_DIRSYNC`
+* `MS_LAZYTIME`
```

Comment:
None of the silently-ignored mount flags are covered by the SCML rules above. In fact, the silently-ignored mount flags of most system calls documented by this PR are not covered by their corresponding SCML rules. Failing doing so would cause a SCML-based, strace log analyzer to think they are unsupported by Asterinas. The SCML rules for `fcntl` do cover the ignored flags; I don't know why you are not doing the same for other system calls.

Reply:
Yes, you are right on this point. Thanks for clarification!